acrobat/scripts/scripts.js

```diff
@@ -10,6 +10,8 @@
  * governing permissions and limitations under the License.
  */
 
+import localeMap from "./maps/localeMap.js";
```

Comment:
```suggestion
import localeMap from './maps/localeMap.js';
```
It should be single quotation for string.